integrationTests/vm/txsFee/utils/utilsESDT.go

```diff
@@ -107,6 +108,40 @@ func CreateESDTTransferTx(nonce uint64, sndAddr, rcvAddr []byte, tokenIdentifier
 	}
 }
 
+type TransferESDTData struct {
```

Comment:
missing comment
Also. let's take this out (and maybe disolve this util package)

Reply:
+1